tests/modules/decrosstalk/test_qc_plotting.py

```diff
@@ -224,28 +258,172 @@ def test_pairwise_plot_generation(tmpdir):
     generate_pairwise_figures(ophys_planes,
                               tmpdir)
 
-    expected_files = []
-    dirname = tmpdir / '1071738390_1071738393_roi_pairs'
-    for fname in ('1080616650_1080616555_comparison.png',
-                  '1080616658_1080616553_comparison.png',
-                  '1080616659_1080616554_comparison.png'):
-        expected_files.append(dirname / fname)
+    expected_files = set()
+    t = pathlib.Path(tmpdir)
+    for plot_path in expected_pairwise:
+        fname = t / plot_path
+        expected_files.add(fname)
 
-    dirname = tmpdir / '1071738394_1071738396_roi_pairs'
-    for fname in ('1080618091_1080616600_comparison.png',
-                  '1080618102_1080616618_comparison.png'):
-        expected_files.append(dirname / fname)
+    for fname in expected_files:
+        if not fname.is_file():
```

Comment:
I think you're testting here that the files were written.
Why not just assert that they `is_file` ?

Reply:
I wanted to customize the error message so we know why the test fails.

Now that I say that, I realize that I could use `assert test_expression, failure_msg` syntax. I'm not used to using assert with that syntax (in a previous life it was drummed into me to use `unittest.assert()` methods, rather than `assert` because they were more informative).

Reply:
as you wish, it just struck me as unusual to raise a `RunTimeError` in a test.

Reply:
Actually, I retract that.
pytest is meant to "provide detailed information on failing assert statements"
https://docs.pytest.org/en/6.2.x/#id1

Reply:
Okay. I've replaced them with `assert finame.is_file()`